ci-operator/jobs/openshift/cluster-baremetal-operator/openshift-cluster-baremetal-operator-master-presubmits.yaml

```diff
@@ -243,8 +243,8 @@ presubmits:
         - --lease-server-credentials-file=/etc/boskos/credentials
         - --report-credentials-file=/etc/report/credentials
         - --secret-dir=/secrets/ci-pull-credentials
-        - --secret-dir=/usr/local/e2e-metal-ipi-cluster-profile
-        - --target=e2e-metal-ipi
+        - --secret-dir=/usr/local/e2e-metal-ipi-ovn-dualstack-cluster-profile
```

Comment:
is somethign alpha sorting in the script maybe?

Reply:
yes, I think so. I got stuck on this for a while in the beginning of my renaming journey. couldn't understand all the wacky changes that 'make jobs' was doing.

should be ok though.